README.md

```diff
@@ -1 +1,20 @@
-# go-auth
\ No newline at end of file
+## Different authentication methods in go
```

Comment:
Ask chat gpt to create a better read me for you

Reply:
READme i will update at the end of project.

Reply:
Abhi se kar de for whatever is done